src/lib/config.ts

```diff
@@ -607,7 +607,7 @@ export class default_config {
         "mkt!": "mktridactylrc -f",
         "mktridactylrc!": "mktridactylrc -f",
         mpvsafe:
-            "js -p tri.excmds.shellescape(JS_ARG).then(url => tri.excmds.exclaim_quiet('mpv ' + url))",
+            "js -p tri.excmds.shellescape(JS_ARG).then(url => tri.excmds.exclaim_quiet('mpv ' + url + ' > /dev/null 2>&1 &'))",
```

Comment:
This presumably will work on OSX and Linux but not Windows.

Could you therefore add a Windows-specific config entry with the previous value? Circa line 1143.

Reply:
Oh… I see that test fail. I will figure out what's wrong and push new changes.

Reply:
I think it's just that colon Vs equals thing.

If you could add a comment above line 609 saying that Windows has an override it'll probably prevent some confusion in future developers. Thanks!